pd/src/wallet/server/thin.rs

```diff
@@ -1,91 +1,23 @@
+use tracing::instrument;
 
 use crate::state;
 
 #[tonic::async_trait]
 impl ThinWallet for state::Reader {
-    type AssetListStream = ReceiverStream<Result<Asset, Status>>;
-
-    #[instrument(skip(self, request))]
+    #[instrument(skip(self, _request))]
     async fn transaction_by_note(
         &self,
-        request: tonic::Request<TransactionByNoteRequest>,
-    ) -> Result<tonic::Response<TransactionDetail>, Status> {
-        self.check_chain_id(&request.get_ref().chain_id)?;
-
-        tracing::debug!(cm = ?hex::encode(&request.get_ref().cm));
-        let state = self.clone();
-        let transaction = state
-            .transaction_by_note(request.into_inner().cm)
-            .await
-            .map_err(|_| tonic::Status::not_found("transaction not found"))?;
-        Ok(tonic::Response::new(transaction))
-    }
-
-    #[instrument(skip(self, request))]
-    async fn asset_lookup(
-        &self,
-        request: tonic::Request<AssetLookupRequest>,
-    ) -> Result<tonic::Response<AssetInfo>, Status> {
-        self.check_chain_id(&request.get_ref().chain_id)?;
-
-        let asset_id = penumbra_crypto::asset::Id::try_from(
-            request
-                .into_inner()
-                .asset_id
-                .ok_or_else(|| tonic::Status::invalid_argument("missing asset id"))?,
-        )
-        .map_err(|_| tonic::Status::not_found("invalid asset ID"))?;
-        tracing::debug!(?asset_id);
-        let state = self.clone();
-        let asset = state
-            .asset_lookup(asset_id)
-            .await
-            .map_err(|_| tonic::Status::not_found("asset not found"))?
-            .ok_or(|| anyhow::anyhow!("asset not found"))
-            .map_err(|_| tonic::Status::not_found("asset not found"))?;
-
-        Ok(tonic::Response::new(asset))
-    }
-
-    #[instrument(skip(self, request))]
-    async fn asset_list(
-        &self,
-        request: tonic::Request<AssetListRequest>,
-    ) -> Result<tonic::Response<Self::AssetListStream>, Status> {
-        self.check_chain_id(&request.get_ref().chain_id)?;
-
-        tracing::debug!("processing request");
-        let state = self.clone();
-
-        let (tx, rx) = mpsc::channel(100);
-        tokio::spawn(
-            async move {
-                let assets = state
-                    .asset_list()
-                    .await
-                    .map_err(|_| tonic::Status::unavailable("database error"))
-                    .unwrap();
-                for asset in &assets[..] {
-tracing::debug!(asset_id = ?hex::encode(&asset.asset_id), asset_denom = ?asset.asset_denom, "sending asset");
-                    tx.send(Ok(asset.clone())).await.unwrap();
-                }
-            }
-            .instrument(Span::current()),
-        );
-
-        Ok(tonic::Response::new(Self::AssetListStream::new(rx)))
+        _request: tonic::Request<NoteCommitment>,
+    ) -> Result<tonic::Response<NoteSource>, Status> {
+        unimplemented!()
```

Comment:
I left this unimplemented since we don't use it and we'll be deleting this code.